freqtrade/rpc/telegram.py

```diff
@@ -113,7 +116,8 @@ def _init_keyboard(self) -> None:
                                  r'/stopbuy$', r'/reload_config$', r'/show_config$',
                                  r'/logs$', r'/whitelist$', r'/blacklist$', r'/bl_delete$',
                                  r'/weekly$', r'/weekly \d+$', r'/monthly$', r'/monthly \d+$',
-                                 r'/forcebuy$', r'/edge$', r'/help$', r'/version$']
+                                 r'/forcebuy$', r'/forcelong$', r'/forceshort$',
+                                 r'/edge$', r'/help$', r'/version$']
```

Comment:
I kinda feel like there should only be a `/forceenter` and `forceexit` just to clean things up, I don't really see the need to separate orders into long and short

Reply:
the problem is - when you need to type this in telegram, it becomes very clumsy to type, so UX is bad.

if you use `/forceenter` - you need another argument (usability of additional arguments suck, as they can be wrong and you won't know until you have to retype it).

so it'd become `/forceenter long XRP/USDT 1.5` .... as opposed to `/forcelong XRP/USDT 1.5` - which is shorter to type (this is something to look out for on phones!).